java/core/src/test/org/apache/orc/impl/TestInStream.java

```diff
@@ -19,6 +19,9 @@
 package org.apache.orc.impl;
```

Comment:
It would be nice to have a test case for blocks that used the original bytes (header.isOriginal).

Reply:
Added the test case